src/components/TableFields.js

```diff
@@ -6,8 +6,8 @@ import './TableFields.css';
 
 class TableFields extends React.Component {
     render() {
-        const draw = this.props.fields.map((chessField) => (
-            <div className="flexChild" key={chessField.id}></div>));
+        const draw = this.props.fields.reverse().map((chessField) => (
+            <div className="flexChild" key={chessField.id}>{chessField.piece}</div>));

```

Comment:
Use unicode for output on the table instead of letters:
White:♙ ♖♘♗♕♔
Black: ♟♜♞♝♛♚